src/app/home/pages/PlansPage/components/PlanStatusIcon.tsx

```diff
@@ -50,6 +53,25 @@ const PlanStatusIcon: React.FunctionComponent<IProps> = ({ plan }) => {
         </span>
       </Popover>
     );
+  } else if (hasDVMBlockedCondition) {
+    const DVMBCondition: ICondition = plan.Migrations.reduce(
```

Comment:
@pranavgaikwad Ideally this logic should exist in a selector but felt beyond the scope of this PR to try to refactor. I would like to build a more refined condition sorting mechanism on the UI side that can put conditions into info/warning/error buckets with prioritization. This will fall into the bucket of work for bringing in a multi-condition view component like we have on the plan validation screen. Similar to what has been done on mtv : [](https://github.com/konveyor/forklift-ui/blob/main/src/app/common/components/StatusCondition.tsx)

Reply:
removed the above logic in favor of encouraging the user to drill down to migration details view.